interfaces/extension.js

```diff
@@ -0,0 +1,17 @@
+type chrome$ViewType = 'tab' | 'popup';
+
+type chrome$extension = {
+  inIncognitoContext: boolean,
+  lastError: ?{message: string},
```

Comment:
Based on https://developer.chrome.com/extensions/extension#property-lastError, seems like this should technically be `lastError: {message: string} | void,`